std/math/algebraic.d

```diff
@@ -43,24 +43,23 @@ import std.traits : CommonType, isFloatingPoint, isIntegral, isSigned, Unqual;
  *     the return type will be the same as the input.
  *
  * Limitations:
- *     Does not work correctly for signed intergal types and value `Num`.min.
+ *     Does not work correctly for value `Num`.min.
```

Comment:
Yes that makes sense.
Would it be proper to explicitly defined this as undefined behaviour as well?
This seems reasonable to me as described below (https://github.com/dlang/phobos/pull/8861#discussion_r1424181108)

Reply:
It can't be undefined behavior without the function becoming `@system`, and we want to keep `abs` `@safe`. You can make it unspecified behavior, but I don't see what's wrong with just telling it overflows and stays the same. Existing code might be checking for it.